nautobot/extras/health_checks.py

```diff
@@ -1,21 +1,56 @@
 """Nautobot custom health checks."""
+from abc import ABC
 from urllib.parse import urlparse
 
 from django.conf import settings
 from django.db import DatabaseError, IntegrityError, connection
 from funcy import omit
 from health_check.backends import BaseHealthCheckBackend
 from health_check.exceptions import ServiceReturnedUnexpectedResult, ServiceUnavailable
+from prometheus_client import Gauge
 from redis import exceptions, from_url
 from redis.client import Redis
 from redis.sentinel import Sentinel
 
 from .models import HealthCheckTestModel
 
 
-class DatabaseBackend(BaseHealthCheckBackend):
+class NautobotHealthCheckBackend(BaseHealthCheckBackend, ABC):
```

Comment:
We don't seem to define any abstract methods. Is there a reason for inheriting from `ABC` anyway?

Reply:
My intent was to mark this ABC as there is the `metric` class attribute you need to set in subclasses. I can either make it an `@abstractproperty` or just omit the entire ABC deal - what are your thoughts?

Reply:
Scratch that, I made the metric optional to deal with cacheops and that resolves this.